app/src/main/java/com/kickstarter/viewmodels/RewardsFragmentViewModel.kt

```diff
@@ -119,6 +184,8 @@ class RewardsFragmentViewModel {
             this.rewardClicked.onNext(reward)
         }
 
+        override fun alertButtonPressed() = this.alertButtonPressed.onNext(null)
```

Comment:
Just curious about sending a null value through this stream, instead of `any`

Reply:
Any implies all possible types (including null), and you don't need to create a new object sending null. I won't mark as incorrect sending an any object though